### PR TITLE
Add wireframe to readme. Add color palette

### PR DESCRIPTION
**Pull request by brobuck87**
Add wireframe and color palette to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,11 @@ A Site dedicated to helping users create the perfect shot of espresso
 - Reddit API to pull top posts from r/coffee
 - Wikipedia API for information about coffee and beans (search)
 - Buyameacoffee API
+
+## Wireframe
+
+![Wireframe](./assets/images/perfect-espresso-tool.drawio.svg)
+
+# Color Palette
```
Comment (johnsickels) [inline README.md]: Should be an h2 ( ## )

**Review by johnsickels — CHANGES_REQUESTED** [review]
Syntax on markdown

File: README.md (hunk 1 above)
Comment (brobuck87) [reply README.md]: All done!

**Review by johnsickels — APPROVED** [review]
Great work!

**Pull request merged**